src/plugins/commander/commands/member.js

```diff
@@ -26,7 +26,11 @@ class MemberCommand extends Command {
     }
 
     getMastheadDiscord(userid) {
-        return got(`https://services.fandom.com/user-attribute/user/${userid}/attr/discordHandle`).json();
+        return got(`https://services.fandom.com/user-attribute/user/${userid}/attr/discordHandle`, {
+            headers: {
+                accept: '*/*'
```

Comment:
This was discussed [here](https://discordapp.com/channels/246075715714416641/476452336282107925/761916568782241802), but to summarise, it appears the response content type is `application/hal+json`. When using Got's `.json()` method, the `Accept` request header is implicitly set to `application/json`, which results in a 406 (on cache miss). Thus, setting the `Accept` request header explicitly to either `application/hal+json` or `*/*` remediates this issue.